### PR TITLE
Fixes #5883 - explain the behavior of -Static for literal types

### LINKED ISSUE #5883: Clarify that -Static with a type rather than an instance reports the type's static members, not those of System.RuntimeType
If you pass a object representing a _type_ (e.g., a type literal such as `[int]`) to `Get-Member`, you get information about the `[System.RuntimeType]` type used to represent types in PowerShell (which derives from `System.Type`) - this behavior is consistent with instances of any other type.

```powershell
PS> ([int] | Get-Member).TypeName[0]
System.RuntimeType  # That is, members of [System.RuntimeType] are being returned.
```

However, when you use `-Static`, you don't get `System.RuntimeType`'s static members, but the static members of the specific type that the `System.RuntimeType` instance represents.

```powershell
PS> ([int] | Get-Member -Static).TypeName[0]
System.Int32  # That is, adding -Static suddenly reports [int] members.
```

This inconsistency may well be an implementation accident rather than by design, but it has always worked this way and should be documented.

Related information:

* https://github.com/PowerShell/PowerShell/issues/6382, a feature request to also allow examining _instance_ members by providing a _type_ as input.

* [This Stack Overflow post](https://stackoverflow.com/q/61584267/45375).

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ef8e6426-4806-2800-5c70-1a9df91a649b
* Version Independent ID: 45960b24-974a-113b-b0df-8055da108b92
* Content: [Get-Member (Microsoft.PowerShell.Utility)](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/get-member?view=powershell-7)
* Content Source: [reference/7.0/Microsoft.PowerShell.Utility/Get-Member.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/7.0/Microsoft.PowerShell.Utility/Get-Member.md)
* Product: **powershell**
* GitHub Login: @JamesWTruher
* Microsoft Alias: **jimtru**

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Fixes [AB#1717885](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1717885) - Fixes #5883 - explain the behavior of -Static for literal types

## PR Context

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [x] Version 7.1 preview content
- [x] Version 7.0 content
- [x] Version 6 content
- [x] Version 5.1 content

**Conceptual content**
- [ ] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles
- [ ] Community content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md [hunk 1]
```diff
@@ -22,167 +21,232 @@ Get-Member [-InputObject <PSObject>] [[-Name] <String[]>] [-MemberType <PSMember
...
 ServiceHandle             Property      System.Runtime.InteropServices.SafeHandle ServiceHandle {get;}
-ServiceName               Property      System.String ServiceName {get;set;}
+ServiceName               Property      string ServiceName {get;set;}
 ServicesDependedOn        Property      System.ServiceProcess.ServiceController[] ServicesDependedOn {get;}
 ServiceType               Property      System.ServiceProcess.ServiceType ServiceType {get;}
 Site                      Property      System.ComponentModel.ISite Site {get;set;}
+StartType                 Property      System.ServiceProcess.ServiceStartMode StartType {get;}
+StartupType               Property      Microsoft.PowerShell.Commands.ServiceStartupType {get;set;}
 Status                    Property      System.ServiceProcess.ServiceControllerStatus Status {get;}
+UserName                  Property      System.String {get;set;}
```
Comment (DCtheGeek) [inline reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md:86]: UserName doesn't exist in 5.1.  I don't have 6 to test if it's there either.  Probably a replicate across the articles that shouldn't be.

File: reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md [hunk 2]
```diff
@@ -22,167 +21,232 @@ Get-Member [-InputObject <PSObject>] [[-Name] <String[]>] [-MemberType <PSMember
...
+### Example 3: Get extended members of service objects
 
-You can use these properties and methods in the same way that you would use an adapted method of the object.
-The second command shows how to display the value of the PSBase property of the Schedule service.
+This example gets the methods and properties of service objects that were extended by using a
+`Types.ps1xml` file or the `Add-Member` cmdlet.
 
-### Example 3: Get extended members of service objects
+```powershell
+Get-Service| Get-Member -View Extended
```
Comment (DCtheGeek) [inline reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md:112]: Space between `Get-Service` and the pipe?

File: reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md [hunk 3]
```diff
@@ -22,167 +21,232 @@ Get-Member [-InputObject <PSObject>] [[-Name] <String[]>] [-MemberType <PSMember
...
 ```
-PS> Get-Service| Get-Member -View Extended
-TypeName: System.ServiceProcess.ServiceController
+
+```Output
```
Comment (DCtheGeek) [inline reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md:115]: On 5.1, I get two additional values, RequiredServices and ToString.

File: reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md [hunk 4]
```diff
@@ -22,167 +21,232 @@ Get-Member [-InputObject <PSObject>] [[-Name] <String[]>] [-MemberType <PSMember
...
+The **MemberType** parameter gets only objects with a value of `NoteProperty` for their
+**MemberType** property.
+
+The command returns the **Message** property of the **EventLogRecord** object.
 
 ### Example 5: Get objects with a specified property
+
+This example gets objects that have a **MachineName** property from a list of cmdlets.
+
+The `$list` variable contains a list of cmdlets to run. The **foreach** statement invokes each
```
Comment (DCtheGeek) [inline reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md:152]: Not sure 'run' is the right word.  Evaluate?  Also, should the **ForEach** be cased to match the example?

File: reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md [hunk 5]
```diff
@@ -22,167 +21,232 @@ Get-Member [-InputObject <PSObject>] [[-Name] <String[]>] [-MemberType <PSMember
...
+command and sends the results to `Get-Member`. The **Name** parameter limits the results from
+`Get-Member` to members that have the name **MachineName**.
+
+```powershell
+$list = "Get-Process", "Get-Service", "Get-Culture", "Get-PSDrive", "Get-ExecutionPolicy"
+ForEach ($cmdlet in $A) {& $cmdlet | Get-Member -Name MachineName}
```
Comment (DCtheGeek) [inline reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md:158]: Shouldn't this be `ForEach ($cmdlet in $list) .....` ?

File: reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md [hunk 6]
```diff
@@ -198,75 +262,29 @@ LastAccessTime
...
-The following list describes the properties that are added when you use the *Force* parameter:
+Adds the intrinsic members and the compiler-generated **get_** and **set_** methods to the display.
+The following list describes the properties that are added when you use the **Force** parameter:
+
+- PSBase: The original properties of the .NET Framework object without extension or adaptation.
+  These are the properties defined for the object class and listed in MSDN.
+- PSAdapted. The properties and methods defined in the PowerShell extended type system.
+- PSExtended. The properties and methods that were added in the Types.ps1xml files or by using the
+  `Add-Member` cmdlet.
+- PSObject. The adapter that converts the base object to a PowerShell**PSObject** object.
```
Comment (DCtheGeek) [inline reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md:280]: Missing space before PSObject.

File: reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md [hunk 7]
```diff
@@ -198,75 +262,29 @@ LastAccessTime
...
-By default, **Get-Member** gets these properties in all views other than Base and Adapted, but it does not display them.
 
-The following list describes the properties that are added when you use the *Force* parameter:
+Adds the intrinsic members and the compiler-generated **get_** and **set_** methods to the display.
+The following list describes the properties that are added when you use the **Force** parameter:
+
+- PSBase: The original properties of the .NET Framework object without extension or adaptation.
+  These are the properties defined for the object class and listed in MSDN.
+- PSAdapted. The properties and methods defined in the PowerShell extended type system.
+- PSExtended. The properties and methods that were added in the Types.ps1xml files or by using the
```
Comment (DCtheGeek) [inline reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md:278]: File should be codefenced.

File: reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md [hunk 8]
```diff
@@ -198,75 +262,29 @@ LastAccessTime
...
+  `Add-Member` cmdlet.
+- PSObject. The adapter that converts the base object to a PowerShell**PSObject** object.
+- PSTypeNames. A list of object types that describe the object, in order of specificity. When
+  formatting the object, PowerShell searches for the types in the Format.ps1xml files in the
+  PowerShell installation directory ($pshome). It uses the formatting definition for the first type
```
Comment (DCtheGeek) [inline reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md:283]: $pshome should be codefenced.

File: reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md [hunk 9]
```diff
@@ -198,75 +262,29 @@ LastAccessTime
...
+The following list describes the properties that are added when you use the **Force** parameter:
+
+- PSBase: The original properties of the .NET Framework object without extension or adaptation.
+  These are the properties defined for the object class and listed in MSDN.
+- PSAdapted. The properties and methods defined in the PowerShell extended type system.
+- PSExtended. The properties and methods that were added in the Types.ps1xml files or by using the
+  `Add-Member` cmdlet.
+- PSObject. The adapter that converts the base object to a PowerShell**PSObject** object.
+- PSTypeNames. A list of object types that describe the object, in order of specificity. When
+  formatting the object, PowerShell searches for the types in the Format.ps1xml files in the
```
Comment (DCtheGeek) [inline reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md:282]: File should be codefenced.

File: reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md [hunk 10]
```diff
@@ -198,75 +262,29 @@ LastAccessTime
...
 
 ### -Force
-Adds the intrinsic members (PSBase, PSAdapted, PSObject, PSTypeNames) and the compiler-generated get_ and set_ methods to the display.
-By default, **Get-Member** gets these properties in all views other than Base and Adapted, but it does not display them.
 
-The following list describes the properties that are added when you use the *Force* parameter:
+Adds the intrinsic members and the compiler-generated **get_** and **set_** methods to the display.
+The following list describes the properties that are added when you use the **Force** parameter:
+
+- PSBase: The original properties of the .NET Framework object without extension or adaptation.
```
Comment (DCtheGeek) [inline reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md:275]: I'd consider making each property bold in the list to help them stand out.

File: reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md (hunk 1 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md:86]: Fixed

File: reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md (hunk 2 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md:112]: Fixed

File: reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md (hunk 3 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md:115]: Fixed

File: reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md (hunk 5 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md:158]: fixed

File: reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md (hunk 4 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md:152]: fixed

File: reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md (hunk 6 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md:280]: Fixed

File: reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md (hunk 7 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md:278]: fixed

File: reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md (hunk 8 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md:283]: Fixed

File: reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md (hunk 9 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md:282]: Fixed

File: reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md (hunk 10 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md:275]: Fixed

**Pull request merged**